streams/src/main/java/org/apache/kafka/streams/kstream/ForeachAction.java

```diff
@@ -27,15 +27,18 @@
  *
  * @param <K> key type
  * @param <V> value type
+ *
  * @see KStream#foreach(ForeachAction)
  */
 public interface ForeachAction<K, V> {
 
     /**
      * Perform an action for each record of a stream.
      *
-     * @param key   the key of the record
-     * @param value the value of the record
+     * @param key
```

Comment:
Is this newline necessary?

Reply:
It's not necessary but make JavaDocs easier to read in general -- and I like to have consistent formatting across the board -- would like to establish this new formatting if nobody objects.

Especially when there is many parameters with name of different length, it cumbersome to get nice formatting -- adding the description in the next line avoid all those whitespaces (and if we rename a parameter it becomes a single line change and no reformatting is necessary)